src/components/label/label.tsx

```diff
@@ -62,6 +62,8 @@ export class Label {
     sourceEvent: MouseEvent;
   }>;
 
+  @Event() calciteInternalLabelConnected: EventEmitter;
```

Comment:
Lets call this `calciteInternalLabelRegister` to be consistent with the other event. 👍

Reply:
Since this event is going to be internal, I think we need to either...

- Mark it as `@internal`
- Use body.dispatchEvent like disconnected event.

Which do you prefer?

Reply:
I think I rather mark it as @internal, since body.dispatchEvent only comes in to solve for not being able to emit on disconnectedCallback.